### DISCUSSION TITLE
Copy http, ftp, command, or on Replit app.

### DISCUSSION THREAD

**Discussion by Zaydiscool777** [community/community#145263] [Repositories]
I was wondering if these shortcuts could be added to the Github app.

**Comment by queenofcorgis** [comment]
@Zaydiscool777 can you share a bit more about your request, any use cases for this, and what you mean by app (desktop or mobile) - it will help our team triage this better!

**Reply by Zaydiscool777** [reply]
I can't quite put it into words, so the best I can do is this: https://u.cubeupload.com/Zaydiscool777/IMG0927.jpeg
